website/pages/docs/nia/installation/install.mdx

```diff
@@ -40,6 +47,22 @@ Usage of consul-terraform-sync:
       Print the version of this daemon.
 ```
 
-# Connect your Network Device
+## Connect your Network Device
+
+Consul-Terraform-Sync interacts with your network device through a network driver. For the Terraform network driver, Consul-Terraform-Sync uses Terraform providers to make changes to your network infrastructure resources. You will need to [search for an existing](/docs/nia/installation/requirements#finding-and-using-terraform-providers) Terraform Provider for each of your network devices or [create a new one](/docs/nia/installation/requirements#how-to-create-a-provider).
```

Comment:
Suggestion to reword the demand into a suggestion: 
> You will need to search for an existing ...

You can reference provider docs on the Terraform Registry to configure each provider or create a new Terraform provider